### ISSUE TITLE
[trigger-sim] remove unused variables (Trac #1137)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1131]
private/trigger-sim/modules/CoincidenceAfterProcessing.cxx contains these variables, which are defined, but not used:
323: signal_particle_id
357: global_readoutwindows
361: global_nugen_flag

signal_particle_id is written to, but never read from. this one raises a compiler warning, the other two are not detected by gcc yet, but may be in the future.

Migrated from https://code.icecube.wisc.edu/ticket/1137
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:03", 
    "description": "private/trigger-sim/modules/CoincidenceAfterProcessing.cxx contains these variables, which are defined, but not used:\n323: signal_particle_id\n357: global_readoutwindows\n361: global_nugen_flag\n\nsignal_particle_id is written to, but never read from. this one raises a compiler warning, the other two are not detected by gcc yet, but may be in the future.", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335643235016", 
    "component": "combo simulation", 
    "summary": "[trigger-sim] remove unused variables", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-17T21:03:23", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150817T21:03:49: hdembinski changed component from "cmake" to "simulation"

**Comment by icecube-trac** [comment]
Trac update at 20150818T20:36:04: hdembinski changed milestone from "" to "IceSim 4.2"

**Comment by icecube-trac** [comment]
Trac update at 20150901T23:57:43: 
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 137159:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137159"
> removed unused variables.  closes #1131.
> ```


* olivas changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:03: 
* olivas changed milestone from "IceSim 4.2" to ""
* olivas commented:

> Milestone IceSim 4.2 deleted
